### ISSUE TITLE
As a taq end-dev, I would like the LIGO plugin to provide a template for a contract

### ISSUE THREAD

**Issue by mweichert** [ecadlabs/taqueria#960]
### Describe the feature you would like

When #915 is implemented, plugin authors can expose named templates (programmable content) that can be instantiated using `taq create`.

This task is to implement a template for the LIGO plugin called _contract_ which when instantiated, creates a LIGO sourcefile in the _contractsDir_ directory.

The _contract_ template should accept one required positional argument called _sourceFileName_. 

In the handler of the template, we need to examine the _sourceFileName_ file extension and determine what syntax is being requested (jsligo, mligo, etc).

The contract itself should have sufficient comments to help someone get started. The increment contract is probably a good example, although I would make the storage a bigmap with metadata as one of its keys that follows the TZIP-16 spec. Again, as there's multiple syntaxes, we need to write the contract in all LIGO syntaxes.

Lastly, the template handler should register the contract using the SDK method implemented in #907 .

### Is this feature solving a problem?

As we now require users to register their contracts, having a template for providing an easy way to create and register a contract at the same time will enhance the end-developer experience.

### Please share any additional details you have?

Although #915 needs to be finished before we can test, as we know the interface, we should be able to develop the template nonetheless.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mweichert** [comment]
Closing, as duplicate of #907.
